COMMERCE-10043 - Headless API Endpoint for adding categories to product fails with NullPointerException

victorcdp opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

matthewchan123 commented:
ci:test:relevant

liferay-continuous-integration commented:
Closing pull request because pulls for reference 7.0.x should not be sent to repository liferay-portal

thektan committed:
LPS-156861 LPS-164409 portal-search-web: Create Vocabulary Layout display template

https://issues.liferay.com/browse/LPS-164409

oliv-yu committed:
LPS-156862 LPS-164846 portal-search-web: Add select vocabularies into category facet configuration

https://issues.liferay.com/browse/LPS-164846

oliv-yu committed:
LPS-156862 LPS-164846 portal-language-lang: Add language keys for select vocabulary configuration

https://issues.liferay.com/browse/LPS-164846

thektan committed:
LPS-156861 LPS-164409 portal-language-lang: Add language keys

https://issues.liferay.com/browse/LPS-164409

thektan committed:
LPS-156861 LPS-164409 portal-language-lang: gradle buildLang

https://issues.liferay.com/browse/LPS-164409

BryanEngler committed:
LPS-156862 LPS-164847 portal-search-api: Add api to set selected vocabulary ids used for filtering

https://issues.liferay.com/browse/LPS-164847

BryanEngler committed:
LPS-156862 LPS-164847 portal-search: Set include regex

https://issues.liferay.com/browse/LPS-164847

BryanEngler committed:
LPS-156862 LPS-164846 portal-search-web: Add widget config to set vocabularies that will be displayed

https://issues.liferay.com/browse/LPS-164846

thektan committed:
LPS-156862 LPS-164846 portal-search-web: Conditionally show info alert using JS

https://issues.liferay.com/browse/LPS-164846

thektan committed:
LPS-156862 LPS-164846 portal-search-web: Rename initialSelectedVocabularyIds for clarity

https://issues.liferay.com/browse/LPS-164846

DiegoHu97 committed:
LPS-161798 add input for date format

thektan committed:
LPS-156862 LPS-164846 portal-search-web: Simplify SELECT_OPTIONS

https://issues.liferay.com/browse/LPS-164846

thektan committed:
LPS-156862 LPS-164846 portal-search-web: Variable name type is at the end

https://issues.liferay.com/browse/LPS-164846

oliv-yu committed:
LPS-156861 LPS-164409 portal-search-web: Remove frequency for vocabulary in template

https://issues.liferay.com/browse/LPS-164409

DiegoHu97 committed:
LPS-161798 retrieve dateFormat value

oliv-yu committed:
LPS-156862 LPS-164846 portal-search-web: Update conditions for displaying info alert

https://issues.liferay.com/browse/LPS-164846

rijema committed:
LPS-163762 Adapting focus on forms according PTR-3342

brianikim committed:
COMMERCE-9999 Show CDNURL images properly for Product Details

Commit:
LPS-161493 Add method to assert batch action in taxonomy vocabulary api marco

Commit:
LPS-161493 Add test on expose batch action for headless admin taxonomy

Commit:
LPS-161493 Add document folder api marco

Commit:
LPS-161493 Add test on expose batch action for headless delivery

brianikim committed:
COMMERCE-9999 Apply Changes to Cart, Mini Cart, and Order Summary

brianikim committed:
COMMERCE-9999 Show CDNURL images properly for Product Publisher

BryanEngler committed:
LPS-156862 LPS-164847 portal-search-api: baseline

https://issues.liferay.com/browse/LPS-164847

BryanEngler committed:
LPS-156862 LPS-164847 portal-search-elasticsearch7-impl: Set include on the aggregation builder

https://issues.liferay.com/browse/LPS-164847

BryanEngler committed:
LPS-156862 LPS-164847 portal-search: Transform selections

https://issues.liferay.com/browse/LPS-164847

brianikim committed:
COMMERCE-9999 Show CDNURL images properly for CPInstance

Commit:
LPS-161493 Update to support get depot id without selenium webdriver

Commit:
LPS-161493 Update tests to get depot id without selenium web driver

Commit:
LPS-161493 Update delete depot and get site id without selenium webdriver

oliv-yu committed:
LPS-156862 LPS-164846 portal-search-web: Disable selector and add help text+link with legacy field

https://issues.liferay.com/browse/LPS-164846

oliv-yu committed:
LPS-156862 LPS-164846 portal-language-lang: Update language keys for vocabulary selector - disabling, alert message

https://issues.liferay.com/browse/LPS-164846

BryanEngler committed:
LPS-156862 LPS-164846 portal-language-lang: gradlew buildLang

https://issues.liferay.com/browse/LPS-164846

oliv-yu committed:
LPS-156862 LPS-164846 portal-search-web: Add support for learn messages inside SelectVocabularies

https://issues.liferay.com/browse/LPS-164846

BryanEngler committed:
LPS-156862 LPS-164846 portal-search-web: Rearrange the select vocab's alert and helptext

https://issues.liferay.com/browse/LPS-164846

oliv-yu committed:
LPS-156862 LPS-164846 portal-language-lang: gradlew buildLang

https://issues.liferay.com/browse/LPS-164846

kellylisbino committed:
LPS-164126 Update Commerce Channel

BryanEngler committed:
LPS-156862 LPS-164848 portal-search-web: Update test

https://issues.liferay.com/browse/LPS-164848

drewbrokke committed:
LPS-163641 account-service: adds new field to service.xml

BryanEngler committed:
LPS-156862 LPS-164850 portal-search-web: Test with both fields

https://issues.liferay.com/browse/LPS-164850

drewbrokke committed:
LPS-163641 account: autogenerated: buildService after add field

Commit:
LPS-163790 Update tests to asset library id instead of group id

ces-totrinh committed:
LPS-163350 Set redirect correctly

4lejandrito committed:
LPS-163574 These are now named types

4lejandrito committed:
LPS-163574 Children are now SchemaElements

4lejandrito committed:
LPS-163574 The name is no longer exposed in the base interface

4lejandrito committed:
LPS-163574 DirectiveWirer no longer exists

I believe this is harmless

4lejandrito committed:
LPS-163574 Using the visitor to mutate the schema

The schema is now immutbale. To mutate it we have to use the
SchemaTransformer. This also means we get rid of some nasty relection
code we had.

See https://www.graphql-java.com/documentation/schema#changing-schema

carolmariaabb committed:
LPS-164927 Remove Feature Flag for LPS-159151/LPS-135404

4lejandrito committed:
LPS-163574 Fixing threadlocals by running in the same thread

See https://github.com/graphql-java-kickstart/graphql-java-servlet/commit/0e5840808ef50f79be75609594a11b12ef01a1fa#diff-a166ddb02fc54c24bbd3a3759107eebb065981ac3824adf877c1e134afd1a3ccR44

ling-alan-huang committed:
LPS-147681 Add javax.portlet.version=3.0

ling-alan-huang committed:
LPS-147681 Add javax.portlet.version=3.0

ling-alan-huang committed:
LPS-147681 Add javax.portlet.version=3.0

CrescenzoRegaLF committed:
COMMERCE-9633 Add check to avoid a NullPointerException

magjed4289 committed:
LPS-163790 Simplify assertion

oliv-yu committed:
LPS-156861 LPS-164409 portal-search-web: Shift vocabulary name to top of panel when single vocab

https://issues.liferay.com/browse/LPS-164409

magjed4289 committed:
LPS-164523 Ignoring ObjectEntryDeletionImpossibleWithPreventDeletionTypeRelationshipCreated until issue fixed

john-co committed:
Revert "LPS-158667 Quarantine failing tests"

This reverts commit 440b9f640ab77bb5c06a1c39d9b607d179ac4c14.

john-co committed:
Revert "LPS-158667 Skip portal instance"

This reverts commit 4d3283b0ef751e5aeba72beb0b76f5901d57577c.

john-co committed:
LPS-159786 Return tests to default upgrade process

john-co committed:
LPS-159786 Refactor status filter add test

CrescenzoRegaLF committed:
COMMERCE-9633 Autogenerated code (blade gw buildService)

CrescenzoRegaLF committed:
COMMERCE-9633 Add CPInstance list as return to buildCPInstances method

CrescenzoRegaLF committed:
COMMERCE-9633 Add updating of price info and price entries after the build of the CPInstances

victorg1991 committed:
LPS-164384 Show display preview when viewing the content in global or in an asset library

4lejandrito committed:
LPS-163574 Upgrading antlr to avoid warning

ANTLR Tool version 4.9.2 used for code generation does not match the current runtime version 4.7.1

Commit:
LRAC-12014 - [F] Create a table to list properties

Commit:
LRAC-12014 - Generated flies

drewbrokke committed:
LPS-163641 account-test: adds test coverage to assert the default value

david-truong committed:
LPS-164600 fix null pointer exception

victorg1991 committed:
LPS-164384 Create component to show the site selector in the display page preview

andrea-ale-sbarra committed:
COMMERCE-10064 fixed validation

andrea-ale-sbarra committed:
COMMERCE-10064 semantic versioning (gw baseLine)

victorg1991 committed:
LPS-164384 AutoSF

drewbrokke committed:
LPS-163641 account-service: sets the default value when adding a new account entry

drewbrokke committed:
LPS-163641 account-service: requires permission check to change the value when using the update method

drewbrokke committed:
LPS-163641 account-service: local service: adds dedicated updateRestrictMembership method

drewbrokke committed:
LPS-163641 account: autogenerated: buildService after adds new local service method

drewbrokke committed:
LPS-163641 account-test: adds test coverage for new update method

drewbrokke committed:
LPS-163641 account-service: remote service: adds dedicated updateRestrictMembership method

drewbrokke committed:
LPS-163641 account: autogenerated: buildService after adds new remote service method

drewbrokke committed:
LPS-163641 account-admin-web: adds an input to toggle membership restriction

drewbrokke committed:
LPS-163641 account-admin-web: unrelated: rely on the portlet framework to handle redirection and exceptions

interaminense committed:
LRAC-12050 Final adjustments

interaminense committed:
LRAC-12050 Generated types

rijema committed:
LPS-163762 Focusing properly in the current Form

shuyangzhou committed:
LPS-165122 Remove UpgradeStepRegistrator.registerInitialDeploymentUpgradeSteps() in favor of "initial.deployment" VerifyProcess

victorg1991 committed:
LPS-164384 Language key

victorg1991 committed:
LPS-164384 buildLang

sergiojimcos committed:
LPS-161883: Delete nestedFields query param in some methods because they don't supported it in Headless Delivery

sergiojimcos committed:
LPS-161883: buildREST

sergiojimcos committed:
LPS-161883: Delete nestedFields query param in all methods because they don't supported it in Headless Admin Taxonomy

drewbrokke committed:
LPS-163641 account-admin-web: row checker: allows checking users if the membership is not restricted

drewbrokke committed:
LPS-163641 account-admin-web: while it should never happen, handle the exception

drewbrokke committed:
LPS-163641 account-service: if the membership is not restricted, allow users with other domains

DiegoHu97 committed:
LPS-161798 adapt mapping input to date input

drewbrokke committed:
LPS-163641 account-admin-web: un-permission the user search so that users who are not yet account members are viewable by an Account Administrator

drewbrokke committed:
LPS-163641 account-admin-web: unrelated: silence unchecked assignment warning

drewbrokke committed:
LPS-163641 portal-language-lang: lang: adds new lang keys

drewbrokke committed:
LPS-163641 account-test: adds coverage to assert allowing membership for users with other domains when membership is not restricted

kellylisbino committed:
LPS-164126 Refactor siteGroupId

kellylisbino committed:
LPS-164126 Rename addCommerceChannel

kellylisbino committed:
LPS-164126 Rename addCommerceCatalogs

kellylisbino committed:
LPS-164126 SF

Commit:
LRAC-12014 - Fix code review issues

Commit:
LRAC-12014 - Generated files

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

ces-binhtran committed:
LPS-164769 Avoid NPE when open notification has deleted user or layout

adolfopa committed:
LPS-164919 Add method to check if there's content for the image

adolfopa committed:
LPS-164919 Tests

adolfopa committed:
LPS-164919 Expose method to check if content exists

adolfopa committed:
LPS-164919 Baseline

adolfopa committed:
LPS-164919 Build service

baharturk committed:
COMMERCE-9633 Add a new test CanAddToCartGeneratedProductSkus

BarbaraCabrera committed:
LPS-164464 Add Rename Action

adolfopa committed:
LPS-164919 Add validation method to check if processing is required

adolfopa committed:
LPS-164919 Reprocess images when entry present but content was lost

drewbrokke committed:
LPS-163641 account-service: adds upgrade process

carolmariaabb committed:
LPS-164928 Remove feature flag for LPS-158962

This reverts commits:

0117e58256b327db76a40317cdb9c6f3508eb49d

ee4bcd743b328c5b385802408d35370d8bafa980

3291fddd8015520c91b5b2adc234d7bb7a4ff5f4

3ab6b38b1851fb7f40fc7874ba6db15e01938e95

IstvanD committed:
LPS-164911 Using the company's default language as a fallback

BarbaraCabrera committed:
LPS-164464 Add Delete Action

magjed4289 committed:
LPS-164895 Moving Headless Team tests to Headless, adding Data Integration ones and placing other headless tests to their correspondent components

magjed4289 committed:
LPS-164895 Changing testray.main.component.name in testcase files

magjed4289 committed:
LPS-164895 Adding missing integration tests to headless suite

DiegoHu97 committed:
LPS-161798 create EDITABLE_TYPE_DEFAULT_OPTIONS constants

BarbaraCabrera committed:
LPS-164464 Add Edit Action

drewbrokke committed:
LPS-163641 account-admin-web: revert removing permission check from search

We will keep this as-is for now until we decide which direction to go to allow
an Account Admin to add users who do not have matching domains.

A couple possible approaches:

1. Un-permission the search, like in the reverted commit
2. Rely on the upcoming "Invite User" feature, so that actually viewing the
   potential users is not necessary.

Revert "LPS-163641 account-admin-web: un-permission the user search so that users who are not yet account members are viewable by an Account Administrator"

This reverts commit 8f0fe4a0479e10b0631a3b85e1a932aeb77d5f23.

patriciajeanperez committed:
LPS-164747 Update tests to use similar but simpler macro to navigate to edit user page

patriciajeanperez committed:
LPS-164747 Delete duplicate of UserNavigator.gotoUser

patriciajeanperez committed:
LPS-164747 Add assertion of Edit User page in one of the tests

ealonso committed:
LPS-164464 Rename

ealonso committed:
LPS-164464 Not needed

ealonso committed:
LPS-164464 Reorder

ealonso committed:
LPS-164464 Rename keys

ealonso committed:
LPS-164464 buildLang

ealonso committed:
LPS-164384 build.gradle

BryanEngler committed:
LPS-156862 LPS-164846 portal-search-web: Add help text and disable selector when legacy field is being used

https://issues.liferay.com/browse/LPS-164846

BryanEngler committed:
LPS-156862 LPS-164848 portal-search-web: Build display contexts

https://issues.liferay.com/browse/LPS-164848

MarinhoFeliphe committed:
LPS-163011 Use the proper map to filterable fields, ES and Relational DB cannot be use the same EntityModel

MarinhoFeliphe committed:
LPS-163011 Adapt ObjectEntryEntityModelTest

MarinhoFeliphe committed:
LPS-163011 Pass EntityModel as argument of PredicateExpressionVisitorImpl to get the proper filterable field

MarinhoFeliphe committed:
LPS-163011 Cover filter by dateCreated and dateModified

ccorreagg committed:
LPS-164972 Add custom fields as expando attributes in the ServiceContext to be set in the REST API

alessiorendina committed:
COMMERCE-9633 Semantic versioning (gw baseline)

sergiojimcos committed:
LPS-161883: Add nestedFields in getSiteContentTemplate query param and fields and restrictFields query params in getSiteWikiPageByExternalReferenceCodeWikiPageExternalReferenceCodeWikiPageAttachmentByExternalReferenceCode method

magjed4289 committed:
LPS-164895 Removing modules/apps/headless/test.properties since all the submodules with tests have their own separate config file

sergiojimcos committed:
LPS-164669: Update createBatch action using the correct one

sandrodw3 committed:
LPS-164759 Adapt getLabel to get localized name

thaiscabrals committed:
LPS-165047 Add Tests Failing in quaratine

thaiscabrals committed:
LPS-165047 Add tests Failing in quaratine

markocikos committed:
LPS-159786 Generate data on the first render

brianchandotcom committed:
LPS-164972 Rename

brianchandotcom committed:
LPS-164972 auto SF

brianchandotcom committed:
LPS-164384 Wordsmith

brianchandotcom committed:
LPS-164384 Regen

adolfopa committed:
LPS-165060 Create README file on store initialization

ealonso committed:
LPS-164759 Also apply here

ealonso committed:
LPS-164759 Removes deprecated classes

brianchandotcom committed:
LPS-165060 Wordsmith

ealonso committed:
LPS-164759 Update tests

joseabelenda committed:
LPS-165052 Fix NPE

ealonso committed:
LPS-164759 baseline

thaiscabrals committed:
LPS-165079 Add Tests Failing in quaratine

Commit:
LRQA-77900 Workflow Quarantined Test

Commit:
LRAC-12014 - Generated files

drewbrokke committed:
LPS-163641 account-api: baseline

Commit:
COMMERCE-10043 - (headless-commerce) - adding endpoint to post new category in a product by the ERC

Commit:
COMMERCE-10043 - (headless-commerce) - buildREST

Commit:
COMMERCE-10043 - (headless-commerce-admin-catalog-impl) - Add implementation method for adding a Category to a Product through the ERC

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

Commit:
LPS-164851 Fix CP2SubscriptionTerms tests

Commit:
LPS-164851 Fix CP2OverviewPage tests

Commit:
LPS-164851 Fix CP2ActivationCardAnalyticsCloud tests

Commit:
LPS-164851 Fix CP2ActivationCardAnalyticsCloud tests

Commit:
LPS-164851 Fix CP2NavigationMenu path

Commit:
LPS-164851 Fix CP2Navigation Menu path

Commit:
LPS-164851 Fix CustomerPortal macro

Commit:
LPS-164851 Fix CP2NavigationMenu macro

Commit:
LPS-164851 Auto SF

shuyangzhou committed:
LPS-165122 Semantic versioning

brianchandotcom committed:
artifact:ignore message-boards-service 5.0.58 prep next

brianchandotcom committed:
LPS-164920 SF

brianchandotcom committed:
COMMERCE-9999 SF

brianchandotcom committed:
LPS-161171 regen

brianchandotcom committed:
POSHI-416 prep next

brianchandotcom committed:
Revert "LPS-164523 Ignoring ObjectEntryDeletionImpossibleWithPreventDeletionTypeRelationshipCreated until issue fixed"

This reverts commit 6d44519576452b0ea710f868a49ec70020c453a9.

brianchandotcom committed:
Revert "LPS-163790 Update tests to asset library id instead of group id"

This reverts commit 221a101367834b4c84f24516d14c4039b68cae49.

brianchandotcom committed:
Revert "LPS-163790 Simplify assertion"

This reverts commit 44030e4d9e57e2dbf02d3178461c572434c4d985.

brianchandotcom committed:
Revert "LPS-161493 Update delete depot and get site id without selenium webdriver"

This reverts commit 19202e1da0dcf46591278b9653923ee760ee0e76.

brianchandotcom committed:
Revert "LPS-161493 Update to support get depot id without selenium webdriver"

This reverts commit 2bd0afa8dd03f49c429696c46ed5ea23abc33119.

brianchandotcom committed:
Revert "LPS-161493 Update tests to get depot id without selenium web driver"

This reverts commit 92a16c95c5d30dbebc3054ffb9e278562e9ab41c.

brianchandotcom committed:
Revert "LPS-161493 Add test on expose batch action for headless delivery"

This reverts commit 4898d1c5993354b6c60533097d62d06c4d78a2a9.

brianchandotcom committed:
Revert "LPS-161493 Add document folder api marco"

This reverts commit bb88c9d2187c6aeadc85104a94d38f7b729efb85.

brianchandotcom committed:
Revert "LPS-161493 Add test on expose batch action for headless admin taxonomy"

This reverts commit f9b3574ffac6dbfa06758bef6df732ac1eea3811.

brianchandotcom committed:
Revert "LPS-161493 Add method to assert batch action in taxonomy vocabulary api marco"

This reverts commit fb959bc7486af2bc36e6f02adf2f32b0b07b2d7e.

brianchandotcom committed:
Revert "LPS-164895 Removing modules/apps/headless/test.properties since all the submodules with tests have their own separate config file"

This reverts commit 1159c5580fe573f9e2e839fcc54a6a5d8f5db483.

brianchandotcom committed:
Revert "LPS-164895 Adding missing integration tests to headless suite"

This reverts commit 022b0e5dd2984241c5ebb8dd75abc70530fdc6be.

brianchandotcom committed:
Revert "LPS-164895 Changing testray.main.component.name in testcase files"

This reverts commit 4e2e595a53d1680d50ac04a44d6c32084a119eed.

brianchandotcom committed:
Revert "LPS-164895 Moving Headless Team tests to Headless, adding Data Integration ones and placing other headless tests to their correspondent components"

This reverts commit 9abf9e212e5e75b97a789a2ff31663934a1d7329.

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

ling-alan-huang committed:
LPS-164760 Auto SF

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

ling-alan-huang committed:
LPS-164760 SF, all methods calling '_getCreateAccountURL' throw 'Exception'

ling-alan-huang committed:
LPS-164760 SF, use portal service reference

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
Revert "LRAC-12014 - Generated files"

This reverts commit dc8bd82892e1f12b977d02dc60617d1411ce0405.

brianchandotcom committed:
Revert "LRAC-12014 - Generated files"

This reverts commit 84966c1aa2b1229910995fbfe3d43075e3a05102.

brianchandotcom committed:
Revert "LRAC-12014 - Fix code review issues"

This reverts commit 20060fe29191db9e64a7fedb181896d215dfc4d0.

brianchandotcom committed:
Revert "LRAC-12014 - Generated flies"

This reverts commit 36879655f9d2907680691fe94a564b30d147c480.

brianchandotcom committed:
Revert "LRAC-12014 - [F] Create a table to list properties"

This reverts commit 07e3764c73bb9265c24afa0010882a8cfb4ebc9f.

brianchandotcom committed:
Revert "LPS-163590 - Add updated types"

This reverts commit 3c20a25c3201b8e688c0598200554dbf73f3d089.

brianchandotcom committed:
Revert "LPS-163590 - Fix merge conflicts"

This reverts commit 103e9198662bdbb93c06ac45b22210056f77e314.

brianchandotcom committed:
Revert "LPS-163590 - SF, all package names should start with @liferay"

This reverts commit 6004ae2622e56c302687b404cf0ec24fe5fd2ae3.

brianchandotcom committed:
Revert "LPS-163590 - SF"

This reverts commit 1ed76d3cc4fe10fa9bc14b1f9fbeeb52b8f465b6.

brianchandotcom committed:
Revert "LPS-163590 - Update to use latest npm-scripts which includes the latest bundler"

This reverts commit 45bab692f21a11e7bd5645eee1cbed1b37e887eb.

brianchandotcom committed:
Revert "LPS-163590 - Normalize dependency versions across DXP"

This reverts commit 4b67e9070f36b51e247eefa2228bb8c884bf8096.

brianchandotcom committed:
Revert "LPS-163590 - Add dependencies back into package.json files"

This reverts commit 9942d2e823ed2d0920fc231ca359bb2acb24ecfa.

brianchandotcom committed:
Revert "LPS-163590 - Add latest bundler config for following the global dependencis list"

This reverts commit 4dfb5ea81fa7e8fa69a276f0a16a3a8ca5624616.

brianchandotcom committed:
Revert "LPS-163590 - disable JSONPackageJSONRedundantDependenciesCheck"

This reverts commit d48a16b1d0e081270c4a7d22ac51158decb24ffd.

brianchandotcom committed:
LPS-163641 match order in tables.sql

brianchandotcom committed:
LPS-156861 Sort

brianchandotcom committed:
LPS-156861 Rename (please double check my manual renames here)

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

ling-alan-huang committed:
LPS-164760 Auto SF

ling-alan-huang committed:
LPS-163641 SF

ling-alan-huang committed:
LPS-165124 Auto SF, SF made the incorrect order

ling-alan-huang committed:
LPS-165124 Fix logic to grab the correct attribute name

ling-alan-huang committed:
LPS-165124 Auto SF

liferay-translation committed:
LPS-77699 Update Translations

ces-binhtran committed:
LPS-164769 SF

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

ling-alan-huang committed:
LPS-164760 Auto SF

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

ling-alan-huang committed:
LPS-164760 Auto SF

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

ling-alan-huang committed:
LPS-164760 Auto SF

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

ling-alan-huang committed:
LPS-164760 Auto SF

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

ling-alan-huang committed:
LPS-164760 Auto SF

ling-alan-huang committed:
LPS-164760 Remvoe unused parameter

ling-alan-huang committed:
LPS-164760 Auto SF

ling-alan-huang committed:
LPS-164760 Make methods private in *ServiceImpl.java

Commit:
LPS-165044 talend-common - remove constraints (this makes no sense - why should any method be excluded from content or response body schema resolution)

Commit:
LPS-165136 Modify locator to make it reusable

Commit:
LPS-165136 Modify macro to adapt new behavior

brianchandotcom committed:
LPS-164769 Only use that method for String

liferay-translation committed:
LPS-77699 Update Translations
